sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -459,12 +586,585 @@ edges.
 
 The `node` class provides the {crs}[common reference semantics].
 
+==== Dynamic Parameters
+
+Dynamic parameters are arguments to a node's command-group which can be updated
+by the user after the node has been added to a graph. They are updated on an
+executable graph which contains the nodes associated with the dynamic parameter
+object.
+
+The type of the underlying object a dynamic parameter represents is set at
+compile time using a template parameter. This underlying memory object can be an
+accessor, a pointer to a USM allocation, scalar passed by value, or a raw byte
+representation of the argument. The raw byte representation is intended to enable
+updating arguments set using
+link:../proposed/sycl_ext_oneapi_raw_kernel_arg.asciidoc[sycl_ext_oneapi_raw_kernel_arg].
+
+Dynamic parameters are registered with nodes in a modifiable graph, with each
+registration associating one or more node arguments to the dynamic parameter
+instance. It is valid for a node argument to be registered with more than one
+dynamic argument instance.
```

Comment:
```suggestion
dynamic parameter instance.
```

Reply:
need a description to say what happens to that node argument if the dynamic parameter instances are all updated.